README.md

```diff
@@ -21,7 +21,8 @@ We run it on Raspbery Pis (ARMv8) as a part of [Umbrel OS](https://github.com/ge
 
 If you're looking to run Umbrel on:
 
-- A Raspberry Pi 3 or 4 (recommended) - [Download Umbrel OS](https://github.com/getumbrel/umbrel-os)
+- A Raspberry Pi 4 with at least 2 Gb RAM - [Download Umbrel OS](https://github.com/getumbrel/umbrel-os) or a 64-bit Linux based system with at least 2 Gb of RAM
+- 1 TB of disk space (for bitcoin blockchain and electrum files)
```

Comment:
This isn't required here as all the requirements/instructions are listed on the umbrel-os.
```suggestion

```

Reply:
What about for installs which don't use umbrel-os